purchases/src/main/kotlin/com/revenuecat/purchases/common/events/BackendStoredEvent.kt

```diff
@@ -73,3 +74,28 @@ internal fun PaywallEvent.toBackendStoredEvent(appUserID: String): BackendStored
         ),
     )
 }
+
+/**
+ * Converts a `PaywallEvent` into a `BackendStoredEvent.Paywalls` instance.
+ *
+ * @receiver The `PaywallEvent` to be converted.
+ * @param appUserID The user ID associated with the event.
+ * @return A `BackendStoredEvent.Paywalls` containing a `BackendEvent.Paywalls`.
+ */
+@OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class)
+internal fun CustomerCenterEvent.toBackendStoredEvent(appUserID: String): BackendStoredEvent {
```

Comment:
We've started adding the `@JvmSynthetic` annotation to internal functions like this so they are not available in Java (internal is kind of ignored in Java :( ). We should add it here, and anywhere else moving forward.